controllers/UserManagement/OrganisationController.js

```diff
@@ -40,6 +38,16 @@ class OrganisationController extends BaseController {
       locals.teams = req.sessionModel.get('teamResults') || [];
       locals.users = req.sessionModel.get('userResults') || [];
       locals.usersMetaData = req.sessionModel.get('usersMetaData') || { total: 0, currentPage: 1, perPage: 20};
+      locals.addTeamAttempt = req.sessionModel.get('addTeamAttempt') || false;
+      locals.addTeamSuccess = req.sessionModel.get('addTeamSuccess') || false;
+      locals.addedTeamName = req.sessionModel.get('addedTeamName') || '';
+      locals.teamExistsError = req.sessionModel.get('teamExistsError') || false;
+      locals.IS_EXTERNAL = process.env.IS_EXTERNAL || 'false';
```

Comment:
I think we should default this to `true` given the differences in functionality between the versions.

Reply:
Done